### ISSUE TITLE
NTR: 3 beta-HSD activity low in fibroblast (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8216]
New term request 3 beta-HSD activity low in fibroblast (elthat_100627174016): 3beta-hydroxy C27-steroid dehydrogenase

**Comment by pnrobinson** [comment]
@MickeySegal
is this the enzyme coded by HSD3B7?
What is the context?

**Comment by MickeySegal** [comment]
Yes: https://omim.org/entry/607765

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000825
- label: Reduced C27 3beta-HSD activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:11067870
- def: Concentration or activity of 3beta-hydroxy-delta5-C27-steroid oxidoreductase (C27 3beta-HSD) as measured in cultured fibroblasts is below the limits of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8216
- synonyms:
